src/azul/__init__.py

```diff
@@ -1071,6 +1071,25 @@ def reindex_sources(self) -> list[str]:
     def google_oauth2_client_id(self) -> Optional[str]:
         return os.environ.get('AZUL_GOOGLE_OAUTH2_CLIENT_ID')
 
+    def provider_tf_fragment(self, env_var: str) -> JSON:
```

Comment:
There should be no TF logic in this module. It should just expose the environment variables.